src/qst_client.cr

```diff
@@ -45,7 +46,7 @@ class Lgwsim::QSTClient
     headers = HTTP::Headers.new
     headers["If-None-Match"] = etag if etag
 
-    response = @client.get("/#{@account}/qst/outgoing.xml?max=100", headers: headers)
+    response = @client.get("/#{@account}/qst/outgoing.xml?max=#{@incoming_batch_size.to_s}", headers: headers)
```

Comment:
```suggestion
    response = @client.get("/#{@account}/qst/outgoing.xml?max=#{@incoming_batch_size}", headers: headers)
```

Crystal will already cast to string because of the interpolation (in a more efficient manner).